tests/baselines/reference/conditionalTypeDiscriminatingLargeUnionRegularTypeFetchingSpeedReasonable.errors.txt

```diff
@@ -0,0 +1,8018 @@
+tests/cases/compiler/conditionalTypeDiscriminatingLargeUnionRegularTypeFetchingSpeedReasonable.ts(8011,26): error TS2797: Expression requires comparison of excessively large union or intersection types.

```

Comment:
I'm pretty sure adding an error into this test kinda invalidates it, since it was here to test that we could do a thing without timing out - not being able to do it _at all_ is kinda strictly worse (and, moreover, means we're no longer testing for a timeout). Is it just a matter of the limit being too low? #37749 made it so this is actually quite cheap for us (and this is just a simplified form of a common reactish pattern), so we shouldn't have an error here.

Reply:
Specifically, regarding the limit, these kinda of errors are "hard blocks" in scenarios which may otherwise work, so we should ensure we only ever issue them them in cases where we'd either time out, stack out, or OOM without the error. If we can process the comparison without one of those three things occuring, we shouldn't be retiring, IMO.